language_selection.md

```diff
@@ -0,0 +1,86 @@
+# Language selection
+
+Many factors influence how a 18F engineers should make technology selections
+for their projects. Here, we discuss recommendations on how to select the
+_language_ used in your projects. This document doesn't offer hard-and-fast
+rules, focusing instead on the aspects which should be balanced against each
+other. Also, we explicitly note that language selection is ultimately the
+decision (and responsibility) of the project team. The guidance below is meant
+to help your team make this decision.
+
+This is a living document. After looking at existing tickets, start an issue
+or pull request in GitHub to suggest changes to these standards.
```

Comment:
this might seem obvious since we're reading this in this repo, but maybe link to the new issue and pr urls here in case someone is reading this rendered elsewhere?